apps/DensifyPointCloud/DensifyPointCloud.cpp

```diff
@@ -180,7 +180,7 @@ bool Initialize(size_t argc, LPCTSTR* argv)
 	OPTDENSE::nOptimize = nOptimize;
 	OPTDENSE::nEstimateColors = nEstimateColors;
 	OPTDENSE::nEstimateNormals = nEstimateNormals;
-	if (!bValidConfig && !OPT::strDenseConfigFileName.IsEmpty())
+	if (!bValidConfig)
```

Comment:
why is that needed? it was added in a recent issue causing crashes in certain scenarios

Reply:
It isn't really - I think your window might have been stale when you looked at this or something. That was in this PR after I merged in develop, but I noticed it and reverted the irrelevant changes.

Reply:
strange, all changes were made on github